src/components/experiment-tracking/run-metadata/run-metadata.js

```diff
@@ -131,9 +146,9 @@ const RunMetadata = ({
                     <span
                       className="details-metadata__title-detail"
                       onClick={() => onTitleOrNoteClick(run.id)}
-                      title={sanitiseEmptyValue(run.title)}
+                      title={sanitiseEmptyValue(getTitleByRunId(run.id))}
```

Comment:
can we have runTitle and runNotes initialized and set a value before returning the html template ? This way we can avoid the function call multiple times in the template.

Reply:
Done.